model/installation/registry.yaml

```diff
@@ -0,0 +1,43 @@
+groups:
+  - id: registry.installation
+    type: attribute_group
+    display_name: Installation Attributes
+    brief: >
+      Describes installation attributes.
+    stability: development
+    attributes:
+      - id: installation.id
+        type: string
+        stability: development
+        brief: >
+          A unique identifier representing the installation of an application on a specific device
+        note: |
+          Its value MUST remain the same between launches of the same installation of an application.
+          On a particular device it MIGHT be the same across different applications of the same vendor, however, it MUST be different across applications of different vendors.
```

Comment:
```suggestion
          On a particular device it could be the same across different applications of the same vendor, however, it MUST be different across applications of different vendors.
```

Reply:
> it MUST be different across applications of different vendors.

is this something in instrumentation control? I don't think it's possible to guarantee global uniqueness

Reply:
Related [comment](https://github.com/open-telemetry/semantic-conventions/pull/1897#discussion_r1950578435).

Reply:
I have removed this line as requested by @LikeTheSalad in the referenced thread.